ui/apps/platform/src/Containers/VulnMgmt/Entity/TableWidgetFixableCves.js

```diff
@@ -41,17 +41,20 @@ const TableWidgetFixableCves = ({ workflowState, entityContext, entityType, name
     let queryVulnsFieldName = 'vulns';
     let queryCVEFieldsName = 'cveFields';
     let queryFragment = VULN_CVE_LIST_FRAGMENT;
+    let vulnType = entityTypes.CVE;
```

Comment:
I'm editing this file in a similar way for my ticket, so let me do a preliminary PR for this and it may make this easier.